### DISCUSSION TITLE
How to change the default CIDR of the solution?

### DISCUSSION THREAD

**Discussion by wchaws** [aws-solutions/centralized-logging-with-opensearch#239] [General]
To change the default CIDR for a solution, you must create a custom CIDR VPC ahead and then deploy CLO main stack based on that.

You can use the following AWS CloudFormation templates to create VPCs with custom CIDR block prefixes and then deploy CLO main stack:

1. Download this [3-layer-subnets.template.json](https://github.com/aws-solutions/centralized-logging-with-opensearch/files/13974602/3-layer-subnets.template.json) template.
2. Go to the AWS CloudFormation console and upload the template you just downloaded.  
3. When prompted, enter the CIDR prefix you want for your VPC. For example, if you enter `10.3`, the VPC CIDR will be `10.3.0.0/16`.  
4. Complete the remaining steps to launch your AWS CloudFormation stack.
5. After the CloudFormation stack has finished deploying, use the following link to create the main CLO stack: https://s3.amazonaws.com/solutions-reference/centralized-logging-with-opensearch/latest/CentralizedLoggingFromExistingVPC.template. Ensure that the public and private subnet parameters reference the previous 3-layer-subnets stack.

**Comment by bryceaug** [comment]
This is now covered in the FAQ section of our Implementation Guide under `Setup and Configuration`
https://docs.aws.amazon.com/solutions/latest/centralized-logging-with-opensearch/frequently-asked-questions.html#setup-and-configuration
